default_metadata/component/autocomplete.json

```diff
@@ -7,6 +7,7 @@
   "name": "component-name",
   "legend": "Question",
   "validation": {
-    "required": true
+    "required": true,
+    "autocomplete_item": true
```

Comment:
i think we should just use `autocomplete` that is the name of the component. also there is never just one item